src/docs/analytics.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Analytics"
+---
+
+This guide steps you through instrumenting your code with Sentry's analytics infrastructure.
```

Comment:
> This guide steps you through instrumenting your code with Sentry's analytics infrastructure.

maybe "This guide steps you through instrumenting your code with Sentry's preferred analytics infrastructure." since we didn't create Redash and/or Datadog? or "...Sentry's choice of ... " idk something like that.

Reply:
@ceorourke or maybe `This guide steps you through instrumenting your code with Sentry's 3rd-party analytics infrastructure.`